.env.dist

```diff
@@ -0,0 +1,23 @@
+# This file is a "template" of which env vars needs to be defined in your configuration or in an .env file
+# Set variables here that may be different on each deployment target of the app, e.g. development, staging, production.
+# https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
+
+###> symfony/framework-bundle ###
+APP_ENV=dev
+APP_DEBUG=1
+APP_SECRET=EDITME
+###< symfony/framework-bundle ###
+
+###> doctrine/doctrine-bundle ###
+# Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
+# For a sqlite database, use: "sqlite:///%kernel.project_dir%/var/data.db"
+# Set "serverVersion" to your server version to avoid edge-case exceptions and extra database calls
+DATABASE_URL=mysql://root@127.0.0.1/sylius_%kernel.environment%
```

Comment:
Maybe we can change the database name to `sylius_demo_%kernel.environment%`? WDYT?

Reply:
Of course we can - however, since `dbname` parameter in this repo used to be `sylius`, what's the improvement in changing the database name?

Reply:
It's not exactly `sylius` database, as in the very nearest future we will have also plugin tables in it 😄 I just think it's good to have a database named after the project it's using it - when you develop shop selling TVs on Sylius your database would be rather `best_tvs_best_very_good` than `sylius` :) But I don't insist